cmd/util/printer/print_legacy.go

```diff
@@ -48,7 +48,7 @@ func PrintJobExecutionLegacy(
 	runtimeSettings *cliflags.RunTimeSettingsWithDownload,
 	apiLegacy *client.APIClient,
 	apiV2 clientv2.API,
-	cfg types.IpfsConfig,
+	cfg types.BacalhauConfig,
```

Comment:
I don't think we need the config here anymore.

Reply:
answered above